### PR TITLE
modified button to open link on next tab

### LINKED ISSUE #4360: [FEATURE] button should open discussion page on next tab
### Description

can we do this, or it is fine as it is?

### Screenshots

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by alokVerma749**
## Fixes Issue

 Example: Closes #4360 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4361"></a>

### CODE REVIEW

**Review by Ananya2001-an — COMMENTED** [review]
Looks good.

**Review by rupali-codes — DISMISSED** [review]
lgtm :)

File: pages/index.js [hunk 1]
```diff
@@ -429,10 +429,11 @@ export default function Home({ total, today }) {
           </div>
         </div>
       </div>
-
+
```
Comment (eddiejaoude) [inline pages/index.js:432]: Not sure why the extra spaces are needed here? Please remove

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**